shell/probe/nocturnal.sh

```diff
@@ -1,30 +1,32 @@
 #!/bin/bash
 
 PRELUDE_API=${PRELUDE_API:="https://api.preludesecurity.com"}
+PRELUDE_DIR=${PRELUDE_DIR:="./cache"}
+PRELUDE_SLEEP=${PRELUDE_SLEEP:=14440}
 
-sys=$(uname -s)-$(uname -m)
-dos=$(echo $sys | tr '[:upper:]' '[:lower:]')
+dos=$(echo $(uname -s)-$(uname -m))
```

Comment:
dont need the `echo` bit

Reply:
why are you keeping the echo ?

Reply:
are you looking at an old commit? there are only 3 echo statements currently in this PR, all debug statements that may remain.

Reply:
good call

Reply:
have you maybe not committed what's on your local?

I'm seeing this
![Screenshot 2023-03-07 at 4 53 31 PM](https://user-images.githubusercontent.com/9056078/223590920-958bd52b-55e1-41aa-b2b3-a2751365b61a.png)